deploy/crds/gcp.managed.openshift.io_projectclaims_crd.yaml

```diff
@@ -1,147 +1,161 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Was curious about setting v1 CRD. AFAIK, we are to default to v1beta1 for time being till we have v3 hives present. Also, need to check with boilerplate team too if SRE operators can use v1 CRD now or not. 

cc: @jharrington22 @2uasimojo

Reply:
Agreed, maybe there is an option to dual version the crds?